### DISCUSSION TITLE
Mini PCIE to PCIE adaptor card for EdgeTpu - Amazon

### DISCUSSION THREAD

**Discussion by FrigateEnthusiast** [blakeblackshear/frigate#6193] [General]
Just wanted to share that I ordered a mini-pcie to pcie adaptor board from Amazon for use with the Coral pcie cards and it is working.
Many of these adaptors don't pass in the full pcie bus and so are a no go.

https://www.amazon.com/gp/product/B07ZGDQLNG/ref=ppx_yo_dt_b_asin_title_o01_s00?ie=UTF8&psc=1

$18.99 - Much cheaper than the Ableconn (Ableconn PEX-MP117 Mini PCI-E to PCI-E Adapter Card ), which is not in stock on Amazon.

So for $24.95 EdgeTpu (Mouser) and $18.99 for this adaptor, you are in business, or can add a second card.

So I have both an Abelconn and this Timack adaptors in my one system.

Make sure you switch the voltage to 3.3 volts. And you have to move the two mounting posts down. I also unscrewed the two wifi antennae connectors from the card edge connector.

Proxmox container config: (I'm not blacklisting in my setup)
lxc.mount.entry: /dev/apex_0 dev/apex_0 none bind,optioanl,create=file
lxc.mount.entry: /dev/apex_1 dev/apex_1 none bind,optional,create=file

Frigate config:
detectors:
coral1:
type: edgetpu
device: pci:1
coral2:
type: edgetpu
device: pci:0

**Comment by lesourcil** [comment]
HI, the page doesnt exist anymore, can you name the model/company of the adapter please !

**Reply by FrigateEnthusiast** [reply]
"Mini PCI-E PCI Express to PCI-E 1x Adapter with SIM Card Slot"

Current brand is "BQZYX+" and they are listed for $29.50 I've seen this card listed by the seller and not really the manufacturer/brand. I bought 2 on Amazon for $4.99 each and those were brand "Mustpoint". Prices seem to be all over the map, whatever the market will bare.

https://www.amazon.com/gp/product/B07ZGHPH7K/ref=ppx_yo_dt_b_asin_title_o03_s00?ie=UTF8&psc=1

Easier route might be to get the M2 version of the TPU from Mouser and then a M2 to PCI adapter.
